recipes/volk/all/conandata.yml

```diff
@@ -2,6 +2,9 @@ sources:
   "1.2.190":
     url: "https://github.com/zeux/volk/archive/1.2.190.tar.gz"
     sha256: "07f03720b8c70a626c98cc9545350538122bca9f853e6ed20ccad5a25d55fa4b"
+  "1.2.182":
+    url: "https://github.com/zeux/volk/archive/d75c007f375f35612dba3de512ac73f10bf9aa0e.zip"
```

Comment:
I don't understand why upstream creates tags so randomly.

Reply:
I just relayed on a commit message "Update to 1.2.182".

Reply:
Why is the version 182 significant?

Reply:
Ah, I guess you're trying to match LunarG SDK releases? It sort of makes sense, but that versioning is also pretty random - you're just seeing two independent processes of "create tags occasionally" diverge.

I can create tags for 182 and 189 to make this easier, but I don't track LunarG releases so I can't promise that this will happen by default in the future - happy to create tags on demand of course.

Reply:
I've added a tag for 1.2.182. I can't actually create a tag for 1.2.189 because volk updated from 185 to 190 in a single jump (with no changes other than version update during that). Hopefully that's not a problem, let me know otherwise and I could make a non-master tag for that out of bound.